app-web/src/components/Cards/Card/ComponentCard.js

```diff
@@ -39,6 +45,11 @@ ComponentCard.propTypes = {
   title: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
   resourcePath: PropTypes.string.isRequired,
+  image: PropTypes.string,
+};
+
+ComponentCard.defaultProps = {
+  image: '',
```

Comment:
Maybe set a default image path here? You could make the default image a configuration item like you did with maxTitleLines, maxDescription lines.

Reply:
yes! Thats a great idea, will be working on that on a new pull request. We are going to first identify what default images to use based on the card type :)

Reply:
The react `<Image>` component will not render a 'broken image' if the image fails to load which is the fall back until the next PR